beacon_node/http_api/src/lib.rs

```diff
@@ -1304,6 +1305,46 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    // POST beacon/blocks
+    let post_beacon_blinded_blocks_ssz =
+        eth_v1
+            .and(warp::path("beacon"))
+            .and(warp::path("blinded_blocks"))
+            .and(warp::path::end())
+            .and(warp::body::bytes())
+            .and(chain_filter.clone())
+            .and(network_tx_filter.clone())
+            .and(log_filter.clone())
+            .and_then(
+                |block_bytes: Bytes,
+                 chain: Arc<BeaconChain<T>>,
+                 network_tx: UnboundedSender<NetworkMessage<T::EthSpec>>,
+                 log: Logger| async move {
+                    let block =
+                        match SignedBeaconBlock::<T::EthSpec, BlindedPayload<_>>::from_ssz_bytes(
+                            &block_bytes,
+                            &chain.spec,
+                        ) {
+                            Ok(data) => data,
+                            Err(e) => {
+                                return Err(warp_utils::reject::custom_server_error(format!(
+                                    "{:?}",
+                                    e
+                                )))
```

Comment:
As mentioned in https://github.com/sigp/lighthouse/pull/4479/files#r1268939832, I think this would be best as a `BAD_REQUEST`.